charts/app-config/values-production.yaml

```diff
@@ -2877,6 +2877,8 @@ govukApplications:
             secretKeyRef:
               name: signon-app-whitehall
               key: oauth_secret
+        - name: GOOGLE_TAG_MANAGER_ID
+          value: GTM-NZG8SF2
```

Comment:
This is the wrong GTM ID. According to the [Trello card](https://trello.com/c/EDrVRCsv/1515-set-up-ga4-property-and-gtm-container) and the [original commit in this Whitehall PR](https://github.com/alphagov/whitehall/commit/c9f8e13ce428b624c2175e417d0d731ffb557813), it should be `GTM-P93SHJ4Z`.

It looks like this one has been copied from the [config for content-data-admin](https://github.com/alphagov/govuk-helm-charts/blob/b36c636e740205dac070909df83465e158baaa99/charts/app-config/values-production.yaml#L457).

Reply:
I've just amended the commit and force pushed. Resolved.